### PR TITLE
[Issue #186] Onboarding Documentation Improvements

### LINKED ISSUE #186: [Task]: Test and update onboarding steps
### Summary

Working through the READMEs and updating as any typos or gaps are noted

### Acceptance criteria

I can get onboarded using the steps in the READMEs within the repo, adding/fixing any outside information I needed to get up and runnning

### PR DESCRIPTION
**Pull request by mdragon**
## Summary
Fixes #186

### CODE REVIEW

File: documentation/api/development.md [hunk 1]
```diff
@@ -19,10 +19,23 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
    curl -sSL https://install.python-poetry.org | python3 -
    ```
 
+   - MD- I ended up accomplishing this via
```
Comment (mdragon) [inline documentation/api/development.md]: @chouinar I got this to work, but maybe not via the best route. Calling ```python3``` didn't seem to honor my env settings?

Comment (chouinar) [reply documentation/api/development.md]: Hmm, not sure why any of these docs are specifically saying `python3` for the commands. That hasn't been needed in years. I believe pyenv just swaps out the `python` command. I think some of these docs could really use another pass, some of this is going to lead to people setting their env up in easy-to-break ways.

I'd strongly recommend only ever letting pyenv manage your python installations as I've found things tend to break if you do it yourself - and in ways that are incredibly difficult to fix.

Comment (chouinar) [reply documentation/api/development.md]: Looking at this a bit more, it seems pyenv does update both the `python` and `python3` command.

Can you explain a bit more about what wasn't getting picked up, or what errors you were seeing?

**Comment by mdragon** [comment]
Rebased after navapbc/simpler-grants-gov#190 merged to fix failing package vulnerability scan

File: documentation/api/development.md [hunk 2]
```diff
@@ -11,17 +11,28 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
...
+
+## Database setup: Run Migrations/Seeds
+
+1. If you haven't done local development before you'll need to execute the migrations and seed the DB with data using the steps in [database-local-usage.md](database/database-local-usage.md)
+
+## OpenSearch setup
 
-4. You'll also need [Docker Desktop](https://www.docker.com/products/docker-desktop/)
+1. Run `make init-opensearch` setup the OpenSearch Container
+2. Run `make populate-search-opportunities` to push data previously seeded in the DB into the search index
```
Comment (chouinar) [inline documentation/api/development.md:33]: Might be worth calling out the `make volume-recreate` command as well. In drops the docker volumes and can delete any data you already have locally. Useful if you want to replace your DB data or your DB migrations are in a weird state.

File: documentation/api/development.md [hunk 3]
```diff
@@ -11,17 +11,28 @@ A very simple [docker-compose.yml](../../docker-compose.yml) has been included t
...
+   - If not, run `pyenv init -` and/or restart your shell to ensure it was run automatically
+3. After installing and activating the right version of Python, install
    [poetry](https://python-poetry.org/docs/#installation) and follow the instructions to add poetry to your path if necessary.
 
    ```bash
    curl -sSL https://install.python-poetry.org | python3 -
    ```
 
-3. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
+4. If you are using an M1 mac, you will need to install postgres as well: `brew install postgresql` (The psycopg2-binary is built from source on M1 macs which requires the postgres executable to be present)
```
Comment (chouinar) [inline documentation/api/development.md]: Ah, this was written a while ago, we're not using psycopg2 anymore, and are using version 3 which is _just_ psycopg.

Comment (mdragon) [reply documentation/api/development.md]: Removed.

File: documentation/api/development.md (hunk 2 above)
Comment (mdragon) [reply documentation/api/development.md:33]: Added reference and cleaned up one later in the file to distinguish this from "clean-volumes"

**Review by chouinar — APPROVED** [review]
LGTM!

**Pull request merged**